### DISCUSSION TITLE
Elastic search client gives error while deploying to vercel

### DISCUSSION THREAD

**Discussion by SandeepDev1** [vercel/vercel#9730] [Help] [answered]
### Problem Description

```
Cannot find module './constants'
Require stack:
- /var/task/node_modules/undici/lib/cookies/parse.js
- /var/task/node_modules/undici/lib/cookies/index.js
- /var/task/node_modules/undici/index.js
- /var/task/node_modules/@elastic/transport/lib/connection/UndiciConnection.js
- /var/task/node_modules/@elastic/transport/lib/connection/index.js
- /var/task/node_modules/@elastic/transport/index.js
- /var/task/node_modules/@elastic/elasticsearch/index.js
- /var/task/elasticSearch/client.js
- /var/task/index.js
- /var/task/___now_launcher.js
- /var/runtime/index.mjs
```

I'm getting this error while deploying to vercel.

When I try to run in local it works perfectly fine. I'm not sure whats wrong.. 😢 😢

**Comment by wiiiimm** [comment]
Please check to ensure you have committed `./constants` into your git repo. It has to be `./constants` and not ./`Constants`.

**Comment by SandeepDev1** [comment, marked as answer]
This issue occurs from the elastic search client specifically from [Elastic-transport](https://github.com/elastic/elastic-transport-js) which imports [undici](https://github.com/nodejs/undici) package. The version 5.5.1 works fine but 5.21.0 gives error in vercel when I deploy but works fine in local. Not sure why.

I edited my package-lock.json file to use 5.5.1 rather than above versions and now it deploys fine.
